asset/js/base.js

```diff
@@ -15,6 +15,13 @@
         '.content h1:not(.content__title), .content h2, .content > h3, .content h4, .content h5'
     );
 
+    // Search modal (search modal is not yet available in the DOM)
+    window.setTimeout(function () {
```

Comment:
This is tricky solution and not guarantee it will work always. I mean, we don't know when exactly search modal will be available in the DOM, probably it depends on internet connection speed.
I would prefer to have here jQuery document ready instead if possible.

Reply:
> …tricky solution…

Right.

> would prefer to have here jQuery document ready instead if possible.

The search is add after the document is "ready".